platform/default/include/mbgl/storage/offline_database.hpp

```diff
@@ -82,10 +81,10 @@ class OfflineDatabase : private util::noncopyable {
 
 private:
     void initialize();
-    void handleError(const mapbox::sqlite::Exception&, const char* action);
+    void handleError(const mapbox::sqlite::Exception&, const char* action, bool resetDB = true);
```

Comment:
I don't think, `bool resetDB = true` is needed

Reply:
Done!